joomla/joomla-cms
# com_tags XML coding standard

#### Description

This PR makes cosmetic changes to **com_tags** XML files to be in compliance with XML coding standards (Check https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md).

Only an extra change has been made: replaced `desc="JGLOBAL_FIELD_CREATED_BY_DESC"` for `description="JGLOBAL_FIELD_CREATED_BY_DESC"`
#### How to test

This is a cosmic change. A simple view of the changes should be enough.
But for test you can check if the tags component work as before.
#### Observations

Any improvements are welcome
